### PR TITLE
Java cc8

### PR DESCRIPTION
**Pull request by wafman**
- [x] Top-level README “Table of Contents” is updated
 - [x] Feature tasks for this challenge are completed
 - [x] Unit tests written and passing
     - [x] “MergedLists” - Expected outcome
     - [x] Expected failure
     - [x] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [x] Link to code
     - [x] Picture of whiteboard

### CODE REVIEW

File: code401Challenges/src/test/java/code401Challenges/linkedlist/LinkedListTest.java [hunk 1]
```diff
@@ -163,4 +164,69 @@ public void testReturnNFromEnd_emptyLinkedList(){
         assertNull("should return null", classUnderTest.returnNfromEnd(0));
     }
 
+    @Test
+    public void testMergeList(){
+        LinkedList classUnderTest = new LinkedList();
```
Comment (evanslaton) [inline code401Challenges/src/test/java/code401Challenges/linkedlist/LinkedListTest.java]: It doesn't look like classUnderTest is ever used
